src/index.js

```diff
@@ -9,7 +9,7 @@ import './styles/styles.css';
 let ctToastCount = 0;
 
 const cogoToast = (text, options) => {
-	let rootContainer = document.getElementById('ct-container');
+	let rootContainer = document.querySelector(options.containerCssSelector) || document.getElementById('ct-container');
```

Comment:
Can we change it to `toastContainerID`, where we could pass the ID instead of a CSS selector

Reply:
yep, makes sense